mirrord/layer/src/lib.rs

```diff
@@ -396,6 +396,10 @@ async fn thread_loop(
                     Some(Ok(message)) => {
                         if let Err(err) = layer.handle_daemon_message(
                             message).await {
+                            if let LayerError::SendErrorConnection(_) = err {
+                                info!("Connection closed by agent");
+                                continue;
```

Comment:
perhaps we can write why was the connection closed or what could be a possibility

Reply:
It's not an error so I think we can leave it as is